[2024-04-07T12:15:25.077+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: defi_data_pipeline.establish_database_conn scheduled__2024-04-07T11:01:00+00:00 [queued]>
[2024-04-07T12:15:25.091+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: defi_data_pipeline.establish_database_conn scheduled__2024-04-07T11:01:00+00:00 [queued]>
[2024-04-07T12:15:25.091+0000] {taskinstance.py:2171} INFO - Starting attempt 5 of 5
[2024-04-07T12:15:25.487+0000] {taskinstance.py:2192} INFO - Executing <Task(_PythonDecoratedOperator): establish_database_conn> on 2024-04-07 11:01:00+00:00
[2024-04-07T12:15:25.505+0000] {standard_task_runner.py:60} INFO - Started process 35623 to run task
[2024-04-07T12:15:25.521+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'defi_data_pipeline', 'establish_database_conn', 'scheduled__2024-04-07T11:01:00+00:00', '--job-id', '382', '--raw', '--subdir', 'DAGS_FOLDER/_etl.py', '--cfg-path', '/tmp/tmpqmph0u4a']
[2024-04-07T12:15:25.529+0000] {standard_task_runner.py:88} INFO - Job 382: Subtask establish_database_conn
[2024-04-07T12:15:25.657+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-04-07T12:15:25.811+0000] {task_command.py:423} INFO - Running <TaskInstance: defi_data_pipeline.establish_database_conn scheduled__2024-04-07T11:01:00+00:00 [running]> on host aa86d21a7ebc
[2024-04-07T12:15:26.207+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='defi_data_pipeline' AIRFLOW_CTX_TASK_ID='establish_database_conn' AIRFLOW_CTX_EXECUTION_DATE='2024-04-07T11:01:00+00:00' AIRFLOW_CTX_TRY_NUMBER='5' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-07T11:01:00+00:00'
[2024-04-07T12:15:26.234+0000] {base.py:83} INFO - Using connection ID 'defi_connection' for task execution.
[2024-04-07T12:15:28.282+0000] {taskinstance.py:2699} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/base.py", line 242, in execute
    return_value = super().execute(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 199, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 216, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/_etl.py", line 87, in establish_database_conn
    conn = hook.get_conn()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/postgres/hooks/postgres.py", line 158, in get_conn
    self.conn = psycopg2.connect(**conn_args)
  File "/home/airflow/.local/lib/python3.8/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: could not translate host name "warehouse" to address: Name or service not known

[2024-04-07T12:15:28.319+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=defi_data_pipeline, task_id=establish_database_conn, execution_date=20240407T110100, start_date=20240407T121525, end_date=20240407T121528
[2024-04-07T12:15:28.389+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 382 for task establish_database_conn (could not translate host name "warehouse" to address: Name or service not known
; 35623)
[2024-04-07T12:15:28.448+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-04-07T12:15:28.479+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
